mcp/1.0/modules/ROOT/pages/mcp-connector-studio.adoc

```diff
@@ -9,6 +9,30 @@
 
 include::connectors::partial$t-studio-partial.adoc[tags=topic-intro]
 
```

Comment:
<img width="374" alt="Screenshot 2025-04-15 at 4 31 16 PM" src="https://github.com/user-attachments/assets/68ca8c2c-b83b-41b9-861c-8a41649e70ca" />
You'll want to include these new sections in this part of the MCP Studio page